server/src/test/scala/org/scassandra/server/cors/CorsSupportTest.scala

```diff
@@ -0,0 +1,58 @@
+package org.scassandra.server.cors
+
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest._
+import org.scassandra.server.ScassandraConfig
+import org.scassandra.server.priming.cors.CorsSupport
+import spray.http._
+import spray.routing.HttpService
+import spray.testkit.ScalatestRouteTest
+
+
+class CorsSupportTest extends FlatSpec
+  with ScalatestRouteTest
+  with HttpService
+  with LazyLogging
+  with CorsSupport
+  with Matchers {
+  def actorRefFactory = system
+
+  val testRoute = path("test") {
```

Comment:
Very nice tests!  They helped me understand `CorsSupport` better.   My only suggestion here would be to also add a test to validate the case where `ScassandraConfig.corsEnabled` is false and ensure the rejected message is propagated.